tests/integration/conditions.py

```diff
@@ -229,7 +229,7 @@ def contains(self):
             with open(filename, "r") as textfile:
                 filetext = textfile.read()
             return self.is_within(filetext)
-        except Exception:  # pylint: disable=W0718
+        except Exception:  # pylint: disable=W0703
```

Comment:
Are there more specific exceptions that could/should be caught here like filenotfound io/etc?  I.e. what's this exception actually meant to handle and why is False always the right return value?

Reply:
I plan to create a branch in the future to eliminate as many of these "pylint: disable" messages as possible (there's a decent amount throughout our codebase) so for now I'm going to leave this as is, but I'll come back to it and clean it up in the future.